src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx

```diff
@@ -37,6 +38,12 @@ export const SimOptionsPage = () => {
         { name: t('Settings.SimOptions.Save'), setting: 'SAVE' },
     ];
 
+    const defaultSideButtons: ButtonType[] = [
+        { name: t('Settings.SimOptions.Captain'), setting: 'Captain', index: 0 },
+        { name: t('Settings.SimOptions.Copilot'), setting: 'Copilot', index: 1 },
+        { name: t('Settings.SimOptions.Both'), setting: 'Both', index: 2 },
+    ];
```

Comment:
Probably should have better keys and names to make it clearer what these do.

Reply:
fixed